packages/expo-screen-capture/src/ScreenCapture.ts

```diff
@@ -8,33 +8,29 @@ const emitter = new EventEmitter(ExpoScreenCapture);
 
 const onScreenshotEventName = 'onScreenshot';
 
+// @needsAudit
 /**
  * Returns whether the Screen Capture API is available on the current device.
  *
- * @returns Async `boolean`, indicating whether the Screen Capture API is available on the current device. Currently this resolves `true` on iOS and Android only.
+ * @returns Async `boolean`, indicating whether the Screen Capture API is available on the current
```

Comment:
Why "Async `boolean`" instead of:
```suggestion
 * @returns A promise that resolves to a `boolean` indicating whether the Screen Capture API is available on the current
```